prow/cmd/generic-autobumper/bumper/bumper.go

```diff
@@ -361,20 +361,55 @@ func cdToRootDir() error {
 	return os.Chdir(d)
 }
 
-func Call(stdout, stderr io.Writer, cmd string, args ...string) error {
-	(&logrus.Logger{
+type callOptions struct {
+	ctx *context.Context
```

Comment:
This does not need to be an explicit pointer I think? `Context` is an interface so this would basically be a double pointer and I'd expect everything to work with simply passing `ctx context.Context` around.